taxi/views.py

```diff
@@ -28,6 +28,21 @@
     return render(request, "taxi/index.html", context=context)
 
 
+@login_required
+def update_drivers(request, pk: int):
```

Comment:
Consider adding error handling for the `Car.objects.get(id=pk)` call to manage cases where the car with the given ID does not exist. This will prevent the view from raising an exception if an invalid ID is provided.